docs/cloud/aws/getting-started/non-cu-boulder/aws-cli-saml2aws-non-cu-boulder.md

```diff
@@ -4,13 +4,13 @@ The **saml2aws** CLI tool is used to login and retrieve AWS temporary credential
 
 ## Prerequisites
 
-1. This guide is only relevant if you are not a member of CU Boulder and require access to manage AWS resources owned by CU Boulder. If you are a member of CU Boulder, please see <a href="./aws-console-access.md" target="_blank">this guide</a> on signing in using your IdentiKey.
+1. This guide is only relevant if you are not a member of CU Boulder and require access to manage AWS resources owned by CU Boulder. If you are a member of CU Boulder, please see [this guide](../aws-cli-saml2aws.html) on signing in using your IdentiKey.
```

Comment:
For internal references, you should not use `.html`, instead, `.md` should be used.

Reply:
ccb0f5c